src/pages/user/order/OrderComplete.tsx

```diff
@@ -1,71 +1,123 @@
+import useRefresh from '@hooks/useRefresh';
 
 const Container = styled.div`
-  width: 100vw;
-  min-height: 100vh;
-  padding: 60px 0 80px;
+  width: 100%;
+  height: 100%;
 `;
 
-const Header = styled.div`
-  background: ${Color.WHITE};
-  position: sticky;
-  top: 0;
-  width: 100vw;
-  padding: 25px;
-  box-sizing: border-box;
-  z-index: 100;
-  ${rowFlex({ justify: 'space-between', align: 'center' })}
+const SubContainer = styled.div`
+  margin-top: 45px;
+  gap: 20px;
+  border-top: 10px solid ${Color.LIGHT_GREY};
+  padding-top: 12px;
+  ${colFlex({ justify: 'center', align: 'center' })}
 `;
 
-const SubContainer = styled.div`
+const SubTitleContainer = styled.div`
+  width: 100%;
+  padding: 0 40px;
+  box-sizing: border-box;
   gap: 10px;
-  ${colFlex({ justify: 'center', align: 'center' })}
+  ${colFlex()};
+`;
+
+const SubTitle = styled.div`
+  font-size: 16px;
+  font-weight: 600;
+`;
+
+const ContentContainer = styled.div`
+  width: 100%;
+  gap: 6px;
+  ${colFlex({ align: 'center' })}
+`;
+
+const ContentTitleContainer = styled.div`
+  width: 300px;
+  ${rowFlex({ justify: 'space-between' })}
+`;
+
+const ContentTitle = styled.div`
+  font-size: 13px;
+  font-weight: 500;
 `;
 
 const ContentBox = styled.div`
   width: 100%;
-  padding: 20px;
-  background: #${Color.HEAVY_GREY};
+  padding: 20px 40px;
   box-sizing: border-box;
-  gap: 4px;
+  background: ${Color.LIGHT_GREY};
   ${colFlex()}
 `;
 
-const ContentTitleLabel = styled(AppLabel)`
-  font-weight: bold;
-  margin-bottom: 3px;
+const ContentText = styled.div`
+  font-size: 13px;
+  font-weight: 500;
+  color: #898989;
 `;
 
-const ProductContainer = styled.div`
-  gap: 4px;
-  padding: 10px 0;
+const OrderProductContainer = styled.div`
+  width: 100%;
+  gap: 10px;
   ${colFlex()}
 `;
 
+const OrderProductRow = styled.div`
+  width: 100%;
+  ${rowFlex({ justify: 'space-between' })}
+`;
+
+const OrderProductText = styled.div`
+  font-size: 13px;
+  font-weight: 500;
+`;
+
+const AccountCopyButton = styled.button`
+  background: transparent;
+  border: 1px solid ${Color.GREY};
+  border-radius: 40px;
+  font-size: 13px;
+  font-weight: 500;
+  padding: 0 10px;
+`;
+
+const DescriptionContainer = styled.div`
+  padding: 20px 0;
+  width: 320px;
+  ${colFlex({ justify: 'center', align: 'center' })};
+`;
+
+const Description = styled.div`
+  font-size: 13px;
+  font-weight: 500;
+  text-align: center;
+  color: #898989;
+  white-space: pre-wrap;
+`;
+
 function OrderComplete() {
   const [accountInfo, setAccountInfo] = useState<string>();
   const [searchParams] = useSearchParams();
   const orderId = searchParams.get('orderId');
   const workspaceId = searchParams.get('workspaceId');
 
-  const { fetchWorkspaceAccount } = useWorkspace();
+  const { fetchWorkspaceAccount, fetchWorkspace } = useWorkspace();
   const { fetchOrder } = useOrder();
+  const { allowPullToRefresh } = useRefresh();
   const setOrderBasket = useSetRecoilState(orderBasketAtom);
```

Comment:
```suggestion
  const resetOrderBasket = useResetRecoilState(orderBasketAtom);
```
위와 같이 `useResetRecoilState`훅을 이용하여 초기화할 수 있어요!

Reply:
어떤걸 초기화 할 수 있는거죠..?? 저는 스크롤 했을 때 새로고침이 가능하게끔 한 코드였습니다!

Reply:
## BEFORE
```js
  const setOrderBasket = useSetRecoilState(orderBasketAtom);

  const resetRecoilState = () => {
    setOrderBasket([]);
  };

  useEffect(() => {
    resetRecoilState();
    fetchOrder(orderId);
    fetchWorkspace(workspaceId);
    allowPullToRefresh();
    getAccountInfo();
  }, []);
```

## AFTER
```js
  const resetOrderBasket = useResetRecoilState(orderBasketAtom);

  useEffect(() => {
    resetOrderBasket();
    fetchOrder(orderId);
    fetchWorkspace(workspaceId);
    allowPullToRefresh();
    getAccountInfo();
  }, []);
```

위와 같이 `resetRecoilState`를 선언하지 않고 바로 useEffect 내부에서 `resetOrderBasket`를 호출하면 좀 더 간편하지 않을까 싶었습니다!

Reply:
아 이전 코드에 다신거였군요 확인했습니다!